### PR TITLE
Add code snippet tags to app.py

### PR DESCRIPTION
**Pull request by Dickson-Mwendia**
## Purpose

* Add code snippet tags for out-of-repo code snippet ingestion

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: src/webapp/app.py [hunk 1]
```diff
@@ -280,18 +286,21 @@ def graph():
...
-        # token
+        # <ms_docref_call_ms_graph>
+        # Simple HTTP Get to graph showing the usage of the retrieved access token
         response = requests.get(
             "https://graph.microsoft.com/v1.0/me",
             headers={"Authorization": f"Bearer {result['access_token']}"},
         ).json()
 
-        # Show the "Graph" view
+        # Show the "Graph" view for all users, unauthenticated or not
```
Comment (ckittel) [inline src/webapp/app.py:296]: That's not true, this is only authenticated users.

File: src/webapp/app.py [hunk 2]
```diff
@@ -241,16 +244,18 @@ def graph():
...
         msal_client = msal.ConfidentialClientApplication(
             app.config.get("CLIENT_ID"),
             authority=app.config.get("AUTHORITY"),
             client_credential=app.config.get("CLIENT_CREDENTIAL"),
             token_cache=token_cache,
             http_cache=http_cache,
         )
+        #</ms_docref_configure_app>
 
+        #<ms_docref_acquire_token>
```
Comment (ckittel) [inline src/webapp/app.py:258]: Style has been adding a space before them, but you have it mixed in here.  Can you make them all with a leading space like this example?

```suggestion
        # </ms_docref_configure_app>

        # <ms_docref_acquire_token>
```

File: src/webapp/app.py [hunk 3]
```diff
@@ -348,3 +363,5 @@ def logout():
         return redirect(url_for("index"))
 
     return app
+
+ # <ms_docref_logout_user>
```
Comment (ckittel) [inline src/webapp/app.py:367]: Typo, also the indent doesn't match the indent of the opening tag.  And they _both_ have extra newlines (only one fixed here, but you can fix the other).

```suggestion
    return ap
    # </ms_docref_logout_user>
```

File: src/webapp/app.py [hunk 4]
```diff
@@ -241,16 +244,18 @@ def graph():
         # user's session. This shouldn't be cached across user sessions.
         http_cache: dict = session.get("msal_http_response_cache", {})
 
-        # Create an MSAL client (using app configuration values) and providing
-        # our token cache.
+        #<ms_docref_configure_app>
+        # Create an MSAL client using the app's configuration values and provide the token cache.
```
Comment (ckittel) [inline src/webapp/app.py:248]: This comment is now too long and violates the standard formatting that gets applied when running `black` (and validated with `pylint`).  Can you reapply formatting for consistency?  Thanks.

**Review by ckittel — CHANGES_REQUESTED** [review]

File: src/webapp/app.py (hunk 4 above)
Comment (Dickson-Mwendia) [reply src/webapp/app.py]: @ckittel sure, I've resolved this in https://github.com/Azure-Samples/ms-identity-docs-code-python/pull/9/commits/1139dd0b17773ce6d4cb0619245e6d5c03166450

**Comment by Dickson-Mwendia** [comment]
Thanks @ckittel for the call-outs. I've resolved the issues raised per your feedback. Let me know if it LGTM

**Review by ckittel — APPROVED** [review]
LGTM!

**Pull request merged**